v3/src/components/case-card/card-view.tsx

```diff
@@ -16,16 +17,31 @@ export const CardView = observer(function CardView({onNewCollectionDrop}: CardVi
   const cardModel = useCaseCardModel()
   const data = cardModel?.data
   const collections = data?.collections
+  const areAllCollectionsSummarized = !!collections?.every(c => cardModel?.summarizedCollections.includes(c.id))
   const rootCollection = collections?.[0]
   const selectedItems = data?.selection
   const selectedItemId = selectedItems && Array.from(selectedItems)[0]
   const selectedItemLineage = cardModel?.caseLineage(selectedItemId)
   const contentRef = useRef<HTMLDivElement>(null)
 
-  const handleSelectCases = (caseIds: string[]) => {
+  const handleSelectCases = (caseIds: string[], collectionId: string) => {
+    cardModel?.setShowSummary(caseIds.length > 1, collectionId)
     data?.setSelectedCases(caseIds)
   }
 
+  const handleSummaryButtonClick = () => {
+    cardModel?.setShowSummary(!areAllCollectionsSummarized)
+  }
+
+  // The first time the card is rendered, summarize all collections unless there is a selection.
+  useEffect(function startWithAllCollectionsSummarized() {
+    if (data?.selection.size === 0) {
+      const allCollections = data?.collections.map(c => c.id) ?? []
+      cardModel?.setSummarizedCollections(allCollections)
+    }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
```

Comment:
I'm not sure this is the best way to do this. I had hoped to set the summarized collections list when the card instance is first created, but couldn't figure out how. I explored using `afterCreate` or `afterAttach` in the model, but `self.data` would always return `undefined`, so the collection information wasn't available at that point. I also looked at `createOrShowTableOrCardForDataset` in case-tile-utils.ts but couldn't get it to work there. Am I missing something regarding those options, or perhaps overlooking another alternative?